### DISCUSSION TITLE
Status of: RAIL support in xrdp v.0.9.x

### DISCUSSION THREAD

**Discussion by conloos** [neutrinolabs/xrdp#2238] [Q&A]
Hi,

what is the status of the RAIL implementation.
Currently i can't open any app directly.

Thanks Frank

**Comment by matt335672** [comment]
Hi Frank,

See #1748 and [this wiki page](/neutrinolabs/xrdp/wiki/RAIL-support-in-xrdp). There's been no change since then.

At the moment the team members are focussing on GFX support and improving sesman, so this isn't going to change for a while I'm afraid, unless we get more effort on the project.

Hope this helps.
